oclmath/fpcontrol.h

```diff
@@ -62,8 +72,10 @@
         _mm_setcsr( *mode );
 #elif defined( __PPC__)
         fpu_control = *mode;
-#elif defined (__arm__)
+#elif defined (__arm__) 
```

Comment:
```suggestion
#elif defined (__arm__)
```
Please, no trailing whitespaces.